docs/index.bs

```diff
@@ -3409,6 +3418,9 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
           1. If running the [=Verify Router Condition=] algorithm with |condition|["{{RouterCondition/not}}"] and |serviceWorker| returns false, return false.
           1. Set |hasCondition| to true.
+      1. If |hasCondition| is true, then:
+          1. Increament |serviceWorker|'s [=service worker/router rule count=] by one.
```

Comment:
It sounds like the router rule count kept on incremented even if verify fails later. i.e. addRoutes() failed.
Also, if or/not is nested, they may cause +1 per nest.  Is it intended?

Reply:
Discussed offline. We don't need it anymore.

We have a rough agreement on the new approach, having two limits and validates them in between the step 5 and 6 in addRoutes().
https://w3c.github.io/ServiceWorker/#dom-installevent-addroutes